sap/5.2/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -0,0 +1,638 @@
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Binary
+|===
+
+=== For Configurations
+* <<sap>>
+
+==== Throws
+* SAP:CONNECTIVITY
+* SAP:INVALID_CACHE
+* SAP:INVALID_INPUT
+* SAP:METADATA_UNAVAILABLE
+* SAP:NOT_FOUND
+* SAP:NOT_SUPPORTED
+* SAP:PARSING
+* SAP:RETRY_EXHAUSTED
+* SAP:SYSTEM_BUSY
+* SAP:TIMEOUT
+
+[[send]]
+=== Send IDoc
+`<sap:send>`
+
+Sends an IDocument to SAP over an RFC. An RFC can be one of two types for IDocuments:
+
+* Transactional (tRFC): A tRFC is a special form of asynchronous Remote Function Call (aRFC). Transactional RFC ensures transaction-like handling of processing steps that were originally handled autonomously. tRFC is an asynchronous communication method that executes the called function module in the RFC server only once, even if the data is sent multiple times due to some network issue. The remote system need not be available at the time the RFC client program is executing a tRFC.  The tRFC component stores the called RFC function, together with the corresponding data, in the SAP database under a unique transaction ID (TID). tRFC is similar to aRFC in that it does not wait at the target system (similar to a registered post). If the system is not available, the data is written into aRFC tables with a transaction ID (SM58) that is chosen by the scheduler RSARFCSE and run every 60 seconds.  tRFC is best used as an extension of asynchronous RFC secure communication between systems.
+
+* Queued (qRFC): A queued RFC is an extension of tRFC. It also ensures that individual steps are processed in sequence. Use qRFC to guarantee that multiple LUWs (Logical Unit of Work transactions) are processed in the order specified by the application. tRFC can be serialized using inbound and outbound queues, hence the name queued RFC (qRFC).  qRFC is best used as an extension of tRFC to define a processing sequence.  Implement qRFC if you want to guarantee that several transactions are processed in a predefined order.
+
+Both tRFC and qRFC have a TID handler.
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| IDoc Name a| String |  The name of the iDocument to execute. |  | x
+| Content a| Binary |  The IDocument to execute. |  #[payload] |
+| Version a| String |  The version on the IDoc. |  |
+| Transaction Id a| String |  The ID that identifies an RFC so it runs only once. |  |
+| Queue Name a| String |  Name of the queue on which the RFC executes. This applies for qRFC since each tRFC call that is processed is given a queue name by the application. A queue name and version are required for each qRFC transaction. |  |
```

Comment:
please add this below:

| IDoc Parsing Options a| <<`IDocParseParameterGroup`>> |  |  |